com.unity.cinemachine/Samples~/3D Samples/Clearshot/LeadShot.unity

```diff
@@ -0,0 +1,3309 @@
+      propertyPath: m_LocalPosition.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalPosition.z
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalRotation.w
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalRotation.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalRotation.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalRotation.z
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.z
+      value: 0
+      objectReference: {fileID: 0}
+    m_RemovedComponents: []
+    m_RemovedGameObjects: []
+    m_AddedGameObjects: []
+    m_AddedComponents: []
+  m_SourcePrefab: {fileID: 100100000, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+--- !u!4 &1009471064 stripped
+Transform:
+  m_CorrespondingSourceObject: {fileID: 4043588081373324795, guid: b67ff370e9ebc4a4abe6732c622975af, type: 3}
+  m_PrefabInstance: {fileID: 1009471063}
+  m_PrefabAsset: {fileID: 0}
+--- !u!1 &1059971139
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 1059971141}
+  - component: {fileID: 1059971140}
+  m_Layer: 0
+  m_Name: Help
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!114 &1059971140
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 1059971139}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 50e0de96d508d8e48b20097039b449d5, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  ButtonText: Help
+  Position: {x: 0.9, y: 0.05}
+  Displayed: 0
+  HelpText: "Sample scene showing Clearshot camera (AlwaysFrameLeader Camera) that
```

Comment:
Instead of "Sample scene showing bla bla bla", say "This scene shows bla bla bla".  It's more grammatically correct.